scripts/js/lib/api/directMapXmlToMdx.ts

```diff
@@ -0,0 +1,127 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2025.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import type {
+  Root,
+  BlockContent,
+  Heading,
+  Text,
+  PhrasingContent,
+  List,
+  ListItem,
+  Paragraph,
```

Comment:
Consider renaming all these types to have Mdast in them, like `TextMdast` or `MdastList`. It can be confusing what is XML versus Mdast. (You'd use `as` keyword)

Reply:
Good call